src/guards.ts

```diff
@@ -1,16 +1,23 @@
-import { FLUENTBIT_REGEX, COMMANDS, type FluentBitSchemaType, EXCLUDED_TAGS } from './constants';
+import {
+  FLUENTBIT_REGEX,
+  COMMANDS,
+  type FluentBitSchemaType,
+  EXCLUDED_TAGS,
+  FLUENTBIT_INCLUDE_REGEX,
+} from './constants';
 
-export const isFluentBit = (config: string) => !!config.match(FLUENTBIT_REGEX);
+export const isFluentBit = (config: string) =>
+  !!(config.match(FLUENTBIT_REGEX) || config.match(FLUENTBIT_INCLUDE_REGEX));
```

Comment:
What does the `!!` syntax do? It looks to be similar to an assertion of `it must`, so the string must match either regex?